rlclientlib/generic_event.cc

```diff
@@ -13,17 +13,17 @@ namespace reinforcement_learning {
     , _objects(std::move(objects))
     , _pass_prob(pass_prob)
     , _content_type(content_type) 
-    , _app_id(app_id) {}
-
+    , _app_id(app_id) 
+    , _number_of_events(0) {}
   generic_event::generic_event(const char* id, const timestamp& ts, payload_type_t type, flatbuffers::DetachedBuffer&& payload, event_content_type content_type, const char* app_id, float pass_prob)
     : _id(id)
     , _client_time_gmt(ts)
     , _payload_type(type)
     , _payload(std::move(payload))
     , _pass_prob(pass_prob)
     , _content_type(content_type) 
-    , _app_id(app_id) {}
-
+    , _app_id(app_id)
+    , _number_of_events(0) {}
```

Comment:
number_of_events as property of event may be a bit unclear - maybe something like event_index?